API.md

```diff
@@ -224,14 +473,28 @@ frisby.create('Ensure response arrives within two seconds')
   .toss()
 ```
 
+### not()
+
+Negates an expect, inverting the logic to expect the opposite, e.g. JSON doesn't match.
```

Comment:
What happens if there are more than one `expect`?

Reply:
Ermm. Dunno...

Reading the code, it sets the mode, so would apply to all expects in this chain. I say all expects. Looking at usages, it's just pathMatch ones - `expectJSON`, `expectJSONTypes`, `expectContainsJSON` and `expectJSONLength`. Won't apply to the other inbuilt ones, or to plugins that don't have explicit support.

I've updated the docs to match, but we might want to revisit this and make it more sturdy at some point.

Reply:
Sounds great.